app/modules/ads/setup.js

```diff
@@ -147,6 +148,13 @@ function setupAdContext(adsContext, instantGlobals, isOptedIn = false) {
     context.set('custom.isCMPEnabled', true);
   }
 
+  const btlConfig = instantGlobals.wgAdDriverBillTheLizardConfig;
+  const insertBeforePath = 'slots.incontent_boxad_1.insertBeforeSelector';
+
+  if (context.get('options.slotRepeater') && billTheLizard.hasAvailableModels(btlConfig || {})) {
```

Comment:
Isn't the call to `billTheLizard.hasAvailableModels` missing project name?

Reply:
It is! Sorry, my bad :< Fixed.